index.html

```diff
@@ -45,6 +45,19 @@ <h1 id="title">
         This is a work on progress.
       </p>
     </section>
+    <section id="introduction">
+      <h2>Introduction
+      </h2>
+      <p>
+        This section is non-normative.
+      </p>
+      <p>
+        The scope of this document is to specify the behavior of the 3D content as part of the
+        [[HTML]] document. 3D content downloaded or obtained by some other mechanism
```

Comment:
I think we just want "document".

```suggestion
        [=document=]. 3D content downloaded or obtained by some other mechanism
```